src/assets/styles/10-settings/_spacing.scss

```diff
@@ -0,0 +1,48 @@
+// ----------------------------------
+// UI Settings
+// Spacing
+// https://zpl.io/bJWJNgW
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+:root {
+  --space-unit: 1rem;
+
+  // --space-xs: 4px
+  // --space-sm: 6px
+  // --space-md: 8px
+  // --space-lg: 16px
+  // --space-xl: 32px
+  --space-xs: calc(0.25 * var(--space-unit));
+  --space-sm: calc(0.375 * var(--space-unit));
+  --space-md: calc(0.5 * var(--space-unit));
+  --space-lg: calc(1 * var(--space-unit));
+  --space-xl: calc(2 * var(--space-unit));
+
+  // --space-sm: 8px
+  // --space-md: 12px
+  // --space-lg: 24px
+  // --space-xl: 48px
+  @include breakpoint(sm) {
+    --space-sm: calc(0.5 * var(--space-unit));
+    --space-md: calc(0.75 * var(--space-unit));
+    --space-lg: calc(1.5 * var(--space-unit));
+    --space-xl: calc(3 * var(--space-unit));
+  }
+
+  // --space-md: 16px
+  // --space-lg: 32px
+  // --space-xl: 64px
+  @include breakpoint(md) {
+    --space-md: calc(1 * var(--space-unit));
+    --space-lg: calc(2 * var(--space-unit));
+    --space-xl: calc(4 * var(--space-unit));
+  }
+}
+
+// USAGE
```

Comment:
Probably a good time to discuss comment formatting... any preferences?

My preferred is:

- Single-line comment `//` for anything ~60 chars and under
```css
.c-component--fixed {
  // when `fixed`, overlay needs to be above header
  z-index: 901;
}
```
- Multi-line block for anything that goes over ~60 characters
```css
/**
 * I am a comment that goes over 60 characters so Chris has
 * studiously created a multi-line block comment. Thanks Chris!
 */
```
- Single-line block for a delimiter
```css
/* Headings */

h1 {
  ...
}

h2 {
  ...
}

h3 {
  ...
}
```

Reply:
Again, happy to approve this PR and deal with this separately if that gets us moving.

Reply:
Sure, let's talk about it offline

Reply:
Let's chat about it offline